nodejs/package.json

```diff
@@ -30,7 +30,7 @@
     "@nestjs/common": "9.4.2",
     "@nestjs/core": "9.4.2",
     "@nestjs/platform-express": "9.4.2",
-    "@razzledotai/sdk": "^0.0.41",
+    "@razzledotai/sdk": "file:/Users/olabodeadedoyin/razzle/razzle-server/sdks/razzle-sdk-node/dist",
```

Comment:
Shouldn't we be pointing to a public version of the dependency?
We might forget to update this when we want to deploy